CHANGELOG.md

```diff
@@ -13,6 +13,8 @@
 ### Fixed
 * [ALFREDAPI-419](https://xenitsupport.jira.com/browse/ALFREDAPI-419): Add urldecoding to deleteAssociation endpoint
 * [ALFREDAPI-422](https://xenitsupport.jira.com/browse/ALFREDAPI-422): Fix totalResults and limits for TDMQ's
+* [ALFREDAPI-427](https://xenitsupport.jira.com/browse/ALFREDAPI-427): Replace terms with property values that will cause Invalid Number Exceptions in Solr 6 with dummy terms in FtsNodeVisitor
```

Comment:
@driesdeproost the changelog items should state what issue this solves (audience:user), not how (audience: developer)

Reply:
done

Reply:
Imo we should drop the "in FtsNodeVisitor", but maybe @kpiot123 or @WimCrols can give their opionion

Reply:
Yes, you can drop that part. People reading this don't know what an FtsNodeVisitor is.

Reply:
Actually, what you see here is the outdated
Have a look at https://github.com/xenit-eu/alfred-api/pull/44/files